skll/experiments.py

```diff
@@ -733,12 +736,14 @@ def _classify_featureset(args):
                                                                    round(best_score, 3)))
         else:
             grid_scores = [None]
+            grid_search_cv_results_dicts = [None]
 
-        # print out the tuned parameters and best CV score
-        param_out = ('{}: {}'.format(param_name, param_value)
-                     for param_name, param_value in
-                     iteritems(learner.model.get_params()))
-        logger.info("Hyperparameters: {}".format(', '.join(param_out)))
+        # print out the tuned parameters
+        if grid_search:
```

Comment:
Is the info not useful when we are not doing grid search?

Reply:
My thought was that the comment didn't really reflect what was happening since the hyperparameters will get get printed out whether they're tuned or not. Let me get rid of the conditional and just change comment, though. I can see why you'd want to know what parameters were used even if they're not being tuned via grid search.